_redirects.yml

```
@@ -594,3 +594,5 @@
   - /go/builders/
 "/build/builders/#selected-builder":
   - /go/builders/selected/
+"https://www.docker.com/build-early-access-program/?utm_campaign=onboard-30-customer-zero&utm_medium=in-product-ad&utm_source=desktop_v4":
+  - /go/build-eap
```

Could you add a trailing slash, just so it's consistent with the other redirects? Thx!

```suggestion
  - /go/build-eap/
```

---

updated!